pom.xml

```diff
@@ -265,6 +266,13 @@
 				<scope>test</scope>
 			</dependency>
 
+			<dependency>
```

Comment:
testcontainer should be added to the poms on a per-module basis, not for the whole project

Reply:
I'm not that accustomed to poms, can i specify the testcontainer-version in the root-pom and use is in a module pom?
Because I want to use everywhere the same Testcontainer-Version.